host_cleanup.sh

```diff
@@ -6,12 +6,12 @@ source common.sh
 # Kill and remove the running ironic containers
 for name in ironic ironic-inspector ; do 
     sudo podman ps | grep -w "$name$" && sudo podman kill $name
-    sudo podman ps --all | grep -w "$name$" && sudo podman rm $name
+    sudo podman ps --all | grep -w "$name$" && sudo podman rm $name -f
 done
 
 # Remove existing pod
 if  sudo podman pod exists ironic-pod ; then
-    sudo podman pod rm ironic-pod
+    sudo podman pod rm ironic-pod -f
```

Comment:
Sorry I should have mentioned we have this in 04_.... also, did you need the -f here also?

Reply:
I didn't to correct the problem I had, but I figured we probably want to unconditionally clean up everything and not let pod-rm fail either, should I take it out?

Reply:
No harm in keeping it I was more thinking we could put the -f into 04_... so that they are consistent.

Reply:
Oh I'm sorry, I misunderstood - it's on both now.